std/uni.d

```diff
@@ -1898,7 +1898,7 @@ alias sharSwitchLowerBound = sharMethod!switchUniformLowerBound;
         assert(equal(arr, [1,2,3]));
     }
 
-    static void destroy(T)(ref T[] arr)
+    @trusted static void destroy(T)(scope ref T[] arr)
```

Comment:
Freeing memory is not safe in general. Move the `@trusted` to the call site where track of the reference is kept.

Reply:
Yes, I agree about alloc, realloc and destroy, but in the end: I didn't change their attribute @trusted, they already were declared that way by original @trusted struct GcPolicy and @trusted struct ReallocPolicy. As I outlined in #6104, I currently won't touch function bodies (as far as possible) and be focussed on function/struct signatures for -dip1000 compliance. Frankly speaking, looking deeper into phobos, I was surprised how much @trusted still is in there, part of which possibly undermining memory safety, thus I will leave Your issue for Dmitry or whoever.
Reviewing my changes, there is only 1 place, where I added @trusted where no one was before: mixin template ForwardStrings function fwdStr. I forgot to mention that.